corehq/apps/domain/deletion.py

```diff
@@ -68,12 +69,13 @@ def execute(self, domain_name):
 
 class ModelDeletion(BaseDeletion):
 
-    def __init__(self, app_label, model_name, domain_filter_kwarg, extra_models=None):
+    def __init__(self, app_label, model_name, domain_filter_kwarg, extra_models=None, delete_kwargs={}):
         models = extra_models or []
         models.append(model_name)
         super(ModelDeletion, self).__init__(app_label, models)
         self.domain_filter_kwarg = domain_filter_kwarg
         self.model_name = model_name
+        self.delete_kwargs = delete_kwargs
```

Comment:
I realize it's not currently mutated, but mutable defaults are a hazard in Python
```suggestion
    def __init__(self, app_label, model_name, domain_filter_kwarg, extra_models=None, delete_kwargs=None):
        models = extra_models or []
        models.append(model_name)
        super(ModelDeletion, self).__init__(app_label, models)
        self.domain_filter_kwarg = domain_filter_kwarg
        self.model_name = model_name
        self.delete_kwargs = delete_kwargs or {}
```